common/app/model/dotcomrendering/MostPopular.scala

```diff
@@ -1,10 +1,16 @@
 package model.dotcomrendering
 
+import com.gu.contentapi.client.utils.format.{ArticleDesign, NewsPillar, StandardDisplay}
+import model.ContentFormat
 import play.api.libs.json._
 
 case class OnwardCollectionResponse(
     heading: String,
     trails: Seq[Trail],
+    // todo: remove default if this pattern is adopted
```

Comment:
`OnwardCollectionResponse` is used elsewhere in the codebase, so if we are removing the defaults here, we'll need to refactor the other uses of this class, so that they aren't missing mandatory fields.